Lib/fontTools/misc/visitor.py

```diff
@@ -0,0 +1,114 @@
+"""Generic visitor pattern implementation for Python objects."""
+
+import enum
+
+
+class Visitor(object):
+
+    defaultStop = False
```

Comment:
would be good to have an explicit test to cover the use case of defaultStop = True